components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx

```diff
@@ -1,6 +1,7 @@
 import './CandidateReportHeaderMobile.scss';
 import React, { FC } from 'react';
-import { Link } from 'react-router-dom';
+//TODO - YL-182
+// import { Link } from 'react-router-dom';
```

Comment:
do we need comments in this PR?

Reply:
Yeah, I think these comments should be here until we figure out the root cause of this issue which is discussed here https://spottedzebra.atlassian.net/browse/YL-182

Reply:
Maybe just clearer comment.